flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/BucketWriter.java

```diff
@@ -67,6 +68,7 @@
   private final CompressionType compType;
   private final ScheduledExecutorService timedRollerPool;
   private final PrivilegedExecutor proxyUser;
+  @SuppressWarnings("squid:S1068")
   private volatile String processId;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "processId" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)